.vscode/tasks.json

```diff
@@ -6,12 +6,12 @@
 		{
 			"label": "build",
 			"type": "npm",
-			"script": "esbuild-watch",
-			"problemMatcher": "$esbuild-watch",
-			"isBackground": true,
+			"script": "watch",
+			"problemMatcher": "$tsc-watch",
```

Comment:
Any differences of note between using esbuild-watch and tsp-watch?

Reply:
ESBuild had a different output from the TypeScript Compiler which is what this PR uses for development builds. So, I had to update this to match. As a side note: this actually works better because `esbuild-watch` never reported issues like `tsc-watch` does. So, you'll get notified if your build fails to compile properly before you launch a debug session.